arm/Microsoft.SignalRService/webPubSub/.parameters/full.parameters.json

```
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "value": "westeurope"
+    },
+    "name": {
+      "value": "fullWebPubSubDeployment"
```

As per our current naming approach

```suggestion
      "value": "<<namePrefix>>-az-pubsub-full-001"
```